README.md

```diff
@@ -1,4 +1,102 @@
-# NCPI FHIR Utility
+# 🔥 NCPI FHIR Utility
+<p align="center">
+  <a href="https://github.com/ncpi-fhir/ncpi-fhir-utility/blob/master/LICENSE"><img src="https://img.shields.io/github/license/ncpi-fhir/ncpi-fhir-utility.svg?style=for-the-badge"></a>
+  <a href="https://circleci.com/gh/ncpi-fhir/ncpi-fhir-utility"><img src="https://img.shields.io/circleci/project/github/ncpi-fhir/ncpi-fhir-utility.svg?style=for-the-badge"></a>
+</p>
 
-Tool-chain for validating a FHIR model (conformance resources), deploying
-the model to a server, and releasing the model.
+A small Python library and CLI that aids in FHIR model development.
+
+Some features are:
+
+1. **Validation** - the CLI wraps the Java-based [HL7 FHIR IG Publisher](https://confluence.hl7.org/display/FHIR/IG+Publisher+Documentation) and provides a more user friendly interface to
+   it. Also adds additional validation to ensure consistency in FHIR resource
+   file names and payloads.
+2. **Publish the model** - POST a directory of FHIR resources to a FHIR server
+3. **JSON/XML conversion** - Convert FHIR resource payloads between JSON and XML
+
+## Installation
+1. Git clone this repository
+
+```bash
+$ git clone git@github.com:ncpi-fhir/ncpi-fhir-utility.git
+$ cd ncpi-fhir-utility
+```
+
+2. Setup a Python virtual environment
+
+```bash
+# Create virtualenv
+$ python3 -m venv venv
+
+# Activate virtualenv
+$ source ./venv/bin/activate
+```
+
+3. Install the necessary requirements
+
+```bash
+$ pip install -e .
+```
```

Comment:
```suggestion
## Install with pip

`pip install git+https://github.com/ncpi-fhir/ncpi-fhir-utility.git`

```

Side note: any package that doesn't work without the -e flag has a broken setup or is missing \_\_init\_\_.py files